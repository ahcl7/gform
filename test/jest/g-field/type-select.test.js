import { generateComponent } from "./common";

describe('GField with type select', function () {
  it('should render correctly', () => {
    const component = generateComponent({
      key: 'sex', type: 'input', inputType: 'select', flex: 'md-6', itemText: 'text', itemValue: 'value', sex: '',
      options: [{ text: 'male', value: 0 }, { text: 'female', value: 1 }]
    });
    expect(component.html()).toMatchSnapshot();
  });
});
