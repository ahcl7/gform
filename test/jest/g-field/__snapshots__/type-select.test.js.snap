// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GField with type select should render correctly 1`] = `
"<div class=\\"g-col md-6 px-2\\">
  <div class=\\"g-autocomplete\\">
    <div>
      <div class=\\"g-tf-wrapper g-tf r\\" style=\\"flex-wrap: wrap;\\">
        <!---->
        <!---->
        <div class=\\"input\\">
          <div class=\\"input-slot\\" style=\\"display: contents;\\"></div> <input autocomplete=\\"off\\" label=\\"Sex\\" type=\\"text\\" class=\\"g-tf-input\\">
          <!---->
        </div> <label class=\\"g-tf-label\\">
          Sex
          <!----></label>
        <!---->
        <div class=\\"g-tf-append__inner\\">
          <!----> <i class=\\"g-icon material-icons g-icon__arrow\\" style=\\"font-size: 24px;\\">arrow_drop_down</i></div>
        <!---->
        <!---->
      </div>
    </div>
  </div>
</div>"
`;
